mip/constraint.go

```diff
@@ -0,0 +1,63 @@
+package mip
+
+// Sense defines the constraint operator between the left-hand-side
+// and the right-hand-side.
+type Sense int64
+
+// Sense of a Constraint.
+const (
+	// LessThanOrEqual used to define a less than equal constraint
+	// 		c, _ := d.NewConstraint(mip.LessThanOrEqual, 123.4)
+	//
+	// 		c.NewTerm(1.0, x)  	 // results in 1.0 * x <= 123.4 in solver
+	LessThanOrEqual Sense = iota
+	// Equal used to define a equal constraint
+	// 		c, _ := d.NewConstraint(mip.Equal, 123.4)
+	//
+	// 		c.NewTerm(1.0, x)  	 // results in 1.0 * x = 123.4 in solver
+	Equal
+	// GreaterThanOrEqual used to define a greater or equal constraint
```

Comment:
```suggestion
	// GreaterThanOrEqual is used to define a greater or equal constraint
```
Similar to above. Here we already had the _or_.